links/build.gradle

```diff
@@ -57,9 +57,11 @@ dependencies {
     compile 'javax.inject:javax.inject:1'
     compile 'org.projectlombok:lombok:1.16.10'
 
+    compile group: 'io.swagger', name: 'swagger-annotations', version: '2.0.0-rc2'
     compile 'io.springfox:springfox-swagger2:2.6.1'
     compile 'io.springfox:springfox-swagger-ui:2.6.1'
 
```

Comment:
'swagger-annotations', version: '2.0.0-rc2' is a release candidate issued to support OpenAPI 3.0’s main features, it has some missing features which will be added in the final release. and since there is no final release yet, we use version 1.5 and springfox-swagger 2 (not OpenAPI 3.0). i'm not sure if we get any problems with version 2.0.0-rc2 but they can happen so i'd be grateful if you get rid of ```compile group: 'io.swagger', name: 'swagger-annotations', version: '2.0.0-rc2'``` and update 'io.springfox:springfox-swagger2:2.6.1' to the latest version, namely https://mvnrepository.com/artifact/io.springfox/springfox-swagger2/2.8.0 like other do. thanks!

Reply:
Hi, this task blocks other tasks, that's why I merge and  instead create another issue